velox/CMakeLists.txt

```diff
@@ -11,7 +11,6 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
```

Comment:
This looks like a diff on account of line spacing. Please can we remove it to clean up the diff.